troposphere/static/js/modals/instance/resume.js

```diff
@@ -1,16 +1,31 @@
 import ModalHelpers from "components/modals/ModalHelpers";
-
 import InstanceResumeModal from "components/modals/instance/InstanceResumeModal";
+
 import actions from "actions";
 
 
 export default {
+    /**
+     * Resume any valid/actionable resources
+     *
+     * The action method handles resources verifying type (via instanceof)
+     * to ensure that multi0ple resources can be handled.
+     *
+     * @param resources - collection of resources to act on
+     */
+    resume: function(resources) {
```

Comment:
I think this scenario highlighted in InstanceActionsAndLinks here was not handled. I realize now that I didn't verify functionality from InstanceDetail view. 

That's probably enough to close this for now ❌